### PR TITLE
Arkalogica

### PR DESCRIPTION
**Pull request by jrandiny**

### CODE REVIEW

File: arkav/arkalogica/views.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+
+class SubmitView(generics.GenericAPIView):
+    serializer_class = AnswerReqSerializer
+    permission_classes = (IsAuthenticated,)
+
+    @swagger_auto_schema(operation_summary='Submit Answer',
+                         responses={200: SubmissionSerializer()})
+    def post(self, request, *args, **kwargs):
+        pass
```
Comment (hashshura) [inline arkav/arkalogica/views.py:55]: Hint: `get_or_create` objek submission, trus `create_or_update` objek answer

File: arkav/settings.py [hunk 2]
```diff
@@ -26,7 +26,7 @@
 SECRET_KEY = os.getenv('SECRET_KEY', '(uc8i&7l81878%b6-x5%$n=0fvfb=rxfkw_+l!od^u#q#=+5_3')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = os.getenv('DEBUG') != 'False'
+DEBUG = os.getenv('DEBUG', False)
```
Comment (hashshura) [inline arkav/settings.py:29]:
```py
DEBUG = os.getenv('DEBUG', 'False') != 'False'
```

File: arkav/arkalogica/services.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                status_code=status.HTTP_400_BAD_REQUEST,
+            )
+        if (timezone.now() > session.end_time):
+            raise ArkavAPIException(
+                detail='This session has been ended.',
+                code='is_ended',
+                status_code=status.HTTP_400_BAD_REQUEST,
+            )
+        Submission.objects.update_or_create(user=requested_user)
+        questions = Question.objects.all()
```
Comment (hashshura) [inline arkav/arkalogica/services.py:31]: Langsung return `session` aja (cek komentar di `views.py`)

File: arkav/arkalogica/views.py (hunk 1 above)
Comment (ravielze) [reply arkav/arkalogica/views.py:55]: ok

**Comment by hashshura** [comment]
LGTM

**Review by hashshura — APPROVED** [review]
LGTM

**Pull request merged**